src/tests/test_sexual_behaviour.py

```diff
@@ -71,18 +95,26 @@ def check_num_partners(row):
 
 def test_num_partners():
     """Check that number of partners are reasonable"""
-    pop_data = Population(size=1000, start_date=date(1989, 1, 1)).data
-    assert(any(pop_data["num_partners"] > 0))
+    pop = Population(size=100000, start_date=date(1989, 1, 1))
+    t1 = time.perf_counter()
```

Comment:
These (along with `import time`) can be removed now, or is performance a concern?

Reply:
Yeah, as discussed, I'd rather leave this a profiler or similar rather than have the timing in the code. If you think they'll be needed immediately, can leave them in and remove them in the refactor in #66.